NuGetCleaner/Program.cs

```diff
@@ -0,0 +1,166 @@
+﻿using System;
+using System.Reflection;
+using System.IO;
+using CommandLine;
+using System.ComponentModel;
+using NuGet.Configuration;
+
+namespace NuGetCleaner
+{
+    class Program
+    {
+
+        public static void Main(string[] args)
+        {
+            var osNameAndVersion = System.Runtime.InteropServices.RuntimeInformation.OSDescription;
+            if (!osNameAndVersion.Contains("Windows"))
+            {
+                Console.WriteLine("This tool is currently only available for Windows, sorry for the inconvenience!\n");
+                return;
+            }
+
+            Parser.Default.ParseArguments<Options>(args)
+                .WithParsed(options => Execute(options));
+        }
+
+        public static void Execute(Options options)
+        {
+            if (CheckDisableLastAccess() != 2)
+            {
+                Console.Write("\nYour Last Access updates are not currently enabled so this tool will not work. \n" +
+                    "To enable Last Access updates, run powershell as administrator and input:\n\n" +
+                    "\tfsutil behavior set disablelastaccess 2\n\n" +
+                    "You may be asked to reboot for the settings change to take effect\n\n" +
+                    "Note: Last usage time of packages will only be tracked once the setting is enabled.\n" +
+                    "As such, last package usage will only be determined from the enable date onward.\n" +
+                    "For further information view documentation at https://github.com/chgill-MSFT/NuGetCleaner \n");
+                return;
+            }
+
+            var settings = Settings.LoadDefaultSettings(".");
+            var gpfPath = SettingsUtility.GetGlobalPackagesFolder(settings);
+
+            int Days = options.Days;
+
+            if (options.DryRun)
```

Comment:
`SearchAndPrint` and `SearchAndDestory` are mostly the same, just the body of the inner most loop is different. I'd put that difference in individual methods and have a single `Search` method that accepts and calls an `Action<string>`. That reduces duplicate code.